### PR TITLE
Minor cleanups

### PR DESCRIPTION
**Pull request by chanseokoh**

### CODE REVIEW

**Comment by chanseokoh** [comment]
Oh, wait. I think it's broken.

**Comment by chanseokoh** [comment]
Should be good now.

**Review by loosebazooka — APPROVED** [review]

File: experimental/dotnet/BUILD [hunk 1]
```diff
@@ -1,11 +1,10 @@
 package(default_visibility = ["//visibility:public"])
 
+load("//base:distro.bzl", "DISTRO_SUFFIXES")
 load("@io_bazel_rules_docker//container:container.bzl", "container_image")
 load("@package_bundle_amd64_debian9//file:packages.bzl", packages_amd64_debian9 = "packages")
 load("@package_bundle_amd64_debian10//file:packages.bzl", packages_amd64_debian10 = "packages")
 
-DISTRO_SUFFIXES = ("_debian9", "_debian10")
-
 DISTRO_PACKAGES = {
```
Comment (loosebazooka) [inline experimental/dotnet/BUILD:8]: should distropackages be built off of distrosuffixes?

Comment (chanseokoh) [reply experimental/dotnet/BUILD]: Ah, good catch. Just realized these can go away too.

**Comment by chanseokoh** [comment]
@loosebazooka tests are passing locally. Can you take another quick look?

**Pull request merged**